iiblib/iib_client.py

```diff
@@ -292,6 +297,43 @@ def wait_for_build(self, build):
                 )
             time.sleep(self.poll_interval)
 
+    def regenerate_bundle(
+        self,
+        bundle_image,
+        organization,
+        raw=False,
+    ):
+        """Regenerate bundle image.
+
+        Args:
+            bundle_image (str)
+                The pull specification of the original bundle image
+                that will be modified.
+            organization (str)
+                The name of the organization the bundle should be
+                regenerated for.
+            raw (bool)
+                Return raw json response instead of model instance
+
+        Returns:
+            `RegenerateBundleModel` or dict
+              if raw == True return dict with json response otherwise
+              return `RegenerateBundleModel` instance.
+        """
+
+        post_data = {
+            "from_bundle_image": bundle_image,
+        }
+        if organization:
```

Comment:
Is organization mandatory parameter for this API endpoint? If yes we can remove this if. 
If it is optional we should make the parameter optional as well. `organization=None` L303

Reply:
Thank you. I missed that. Fixed